kratos/tests/cpp_tests/input_output/test_stl_io.cpp

```diff
@@ -128,13 +128,13 @@ KRATOS_TEST_CASE_IN_SUITE(WriteTriangleToSTL, KratosCoreFastSuite)
     KRATOS_CHECK_EQUAL(r_output_model_part.NumberOfProperties(), 1);
     KRATOS_CHECK_EQUAL(r_output_model_part.NumberOfSubModelParts() ,1);
     KRATOS_CHECK_EQUAL(r_output_model_part.NumberOfNodes(), 3);
-    KRATOS_CHECK_EQUAL(r_output_model_part.NumberOfElements(), 1);
+    KRATOS_CHECK_EQUAL(r_output_model_part.NumberOfGeometries(), 1);
     KRATOS_CHECK_EQUAL(r_output_model_part.NumberOfConditions(), 0);
 
     KRATOS_CHECK_EQUAL(r_output_model_part.GetSubModelPart("Main").NumberOfNodes(), 3);
-    KRATOS_CHECK_EQUAL(r_output_model_part.GetSubModelPart("Main").NumberOfElements(), 1);
+    KRATOS_CHECK_EQUAL(r_output_model_part.GetSubModelPart("Main").NumberOfGeometries(), 1);
```

Comment:
```suggestion
    KRATOS_CHECK_EQUAL(r_output_model_part.GetSubModelPart("Main").NumberOfGeometries(), 1);
    KRATOS_CHECK_EQUAL(r_output_model_part.GetSubModelPart("Main").NumberOfElements(), 0);
```

same for the other places